llama-index-integrations/readers/llama-index-readers-kibela/llama_index/readers/kibela/base.py

```diff
@@ -3,13 +3,12 @@
 
 from llama_index.core.readers.base import BaseReader
 from llama_index.core.schema import Document
-from pydantic import BaseModel, parse_obj_as
-from pydantic.generics import GenericModel
+from pydantic import BaseModel, TypeAdapter
```

Comment:
It'd be better to use from our bridge:

```python
from llama_index.core.bridge.pydantic import BaseModel, TypeAdapter
```

You may need to add `TypeAdapter` import to that bridge as well it seems tho.

Reply:
Hi nerdai-san. thanks for review!
I see, it appears that a Bridge has been added to the main llama for compatibility purposes. Thank you for teaching me.
Since TypeAdapter can only be used with Pydantic V2, from a compatibility standpoint, it's better to work with V1. Therefore, I addressed this by adding parse_obj_as used in the original code to the Bridge.